sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
+  if array == nil || array.length <= 1
+    return array # nothing to sort
```

Comment:
consider: catching the error that results from the array containing string and number elements